python/lsst/dax/obscore/siav2.py

```diff
@@ -0,0 +1,703 @@
+# This file is part of dax_obscore.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import annotations
+
+__all__ = ["SIAv2Handler", "siav2_query", "siav2_query_from_raw"]
```

Comment:
Could we also add SIAv2Parameters to __all__ here as I'll be using it in the FastAPI app?

```suggestion
__all__ = ["SIAv2Handler", "SIAv2Parameters", "siav2_query", "siav2_query_from_raw"]
```